src/test/java/myfirstproject/service/impl/PrepareDataImplTest.java

```diff
@@ -0,0 +1,33 @@
+package myfirstproject.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import myfirstproject.model.Fruit;
+import myfirstproject.service.PreparingData;
+import org.junit.Test;
+
+public class PrepareDataImplTest {
+    @Test
+    public void isEmptyData_Ok() {
+        PreparingData preparingData = new PrepareDataImpl();
```

Comment:
let's create field with type `PreparingData` and initialise it inside method annotated with `BeforeClass`
also, what about the name of method?
check [this message](https://mateacademy.slack.com/archives/C040S2B99FH/p1667833485897189)